cmd/operator-sdk/build/cmd.go

```diff
@@ -34,17 +34,19 @@ var (
 	imageBuildArgs string
 	imageBuilder   string
 	goBuildArgs    string
+	skipImage      bool
 )
 
 func NewCmd() *cobra.Command {
 	buildCmd := &cobra.Command{
-		Use:   "build <image>",
+		Use:   "build [<image>]",
 		Short: "Compiles code and builds artifacts",
 		Long: `The operator-sdk build command compiles the Operator code into an executable binary
 and generates the Dockerfile manifest.
 
 <image> is the container image to be built, e.g. "quay.io/example/operator:v0.0.1".
-This image will be automatically set in the deployment manifests.
+By default, this image will be automatically set in the deployment manifests. Note that you can use
+the flag --skip-image in the case you are looking to only compile and prepare the Dockerfile.
```

Comment:
Just a suggestion for rewording the flag help text to clarify what it does. Here and below.
`prepare the Dockerfile` makes it seem like `--skip-flag` is affecting the Dockerfile in some way.
```suggestion
the flag --skip-image to skip building the container image and only build the operator binary.
```